app/assets/javascripts/views/history_view.js

```diff
@@ -5,41 +5,75 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addActivities', 'connectionError');
 
-    this.currentStory = {};
-
-    this.$el.html(this.template({id: 'history', name: I18n.t('projects.show.history')}));
+    this.$el.html(this.template({
+      id: 'history',
+      className:'historycolumn',
```

Comment:
But why is it necessary to have _two_ classes? I think that's the main question, why isn't it possible to use the currently existent `history_column` for what we want to achieve?